package.json

```
@@ -19,8 +19,8 @@
     "@asyncapi/avro-schema-parser": "^0.2.0",
     "@asyncapi/converter": "^0.4.3",
     "@asyncapi/generator": "^1.6.10",
-    "@asyncapi/html-template": "^0.18.2",
-    "@asyncapi/markdown-template": "^0.13.1",
+    "@asyncapi/html-template": "^0.20.0",
+    "@asyncapi/markdown-template": "^0.13.10",
```

We shouldn't be using numbers here but words instead, like this:

```suggestion
    "@asyncapi/markdown-template": "^zero point thirteen point ten",
```

---

:trollface:

---

:trollface: